### ISSUE TITLE
3.6.1.7 Landscape mode on iPad; NTTE info

### ISSUE THREAD

**Issue by edmiller47** [dasilva333/TowerGhostForDestiny#473]
When opening Tower Ghost on iPad while the tablet is in landscape mode, the application displays a white screen with a small black vertical stripe along the left margin.

Opening Tower Ghost in portrait mode works normally. Opening in portrait mode, then rotating the iPad to landscape mode also works as expected.

One other unrelated bug - hovering over "No Time to Explain" on any platform doesn't display the data for the gun, just says "Loading".

**Comment by dasilva333** [comment]
Duplicate of issue #471 also NTTE bug is out of my control, waiting on DestinyDb to update their side. please leave any feedback there thanks

**Comment by edmiller47** [comment]
Sorry for the duplicate bug report of #471. NTTE bug is minor but I hadn't
seen it mentioned.
